tests/Core/Dialog/FluentDialogServiceTests.razor

```diff
@@ -23,4 +23,26 @@
         // Assert
         cut.Verify();
     }
+
+    [Fact]
```

Comment:
Probably interesting to add a second test with a specific Height (ex: `Height="240px").

Reply:
Yes, will do

Reply:
The test above that one already has a height set